api-docs/events.md

```diff
@@ -47,7 +47,21 @@ GET /api/events/:id
     "lastName": "Denning",
     "fullName": "Joel Denning",
     "timestamp": "2019-05-06T06:00:00.000Z"
-  }
+  },
+  "leads": [
+    {
+      "leadId": 1,
```

Comment:
I opted for `leadId` here instead of just `id`. I'm happy to change it if you don't like it.